suite-native/device/src/selectors.ts

```diff
@@ -255,3 +261,57 @@ export const selectShouldFactoryResetBeVisible = createMemoizedSelector(
     (isDeviceInBootloader, hasDeviceFirmwareInstalled) =>
         isDeviceInBootloader && hasDeviceFirmwareInstalled,
 );
+
+export const selectIsDeviceCompromised = createMemoizedSelector(
```

Comment:
These two selectors are great candidates for unit tests.

Reply:
As discussed, I'd avoid doing test for this at this point as it's just moving code from elsewhere. We can do it as a follow up if needed, but since it wasn't tested up to this point, I don't want to bloat the PR.